BUILDING.md

```diff
@@ -59,6 +59,11 @@ note1 - The gcc4.8-libs package needs to be installed, because node
   by Joyent. SmartOS images >= 16.4 are not supported because
   GCC 4.8 runtime libraries are not available in their pkgsrc repository
 
+note2 - Running Node.js on Windows is officially supported using Cmd. Running Node.js
+  through any other Windows shell (e.g. WSL, Git Bash, Cygwin, etc) is experimental.
+  Please note that running Node.js through Git Bash and Cygwin requires usage of [winpty](https://github.com/rprichard/winpty) (installed with Git for Windows) for Node.js to work
+  correctly (e.g. `winpty node.exe script.js`).
```

Comment:
I'd add something like:

>If you call node without the `.exe` extension, `winpty` is used automatically.

Reply:
@gibfahn that'd only be within git for windows, right?

Reply:
Yes.

BTW `Git for Windows` === `Git Bash`, so maybe just use the latter for clarity.

Reply:
@gibfahn alright, got it. let me add that..